packages/linter/src/index.ts

```diff
@@ -30,33 +30,34 @@ import { HeroImageIsDefined } from "./rules/HeroImageIsDefined";
 import { FastGoogleFontsDisplay } from "./rules/FastGoogleFontsDisplay";
 import { GoogleFontPreconnect } from "./rules/GoogleFontPreconnect";
 import { BoilerplateIsRemoved } from "./rules/BoilerplateIsRemoved";
-import { AmpImgUsesSrcSet } from './rules/AmpImgUsesSrcSet';
+import { AmpImgUsesSrcSet } from "./rules/AmpImgUsesSrcSet";
 import { RuleConstructor } from "./rule";
 import { isArray } from "util";
+import * as cheerio from "cheerio";
 
 export enum LintMode {
   Amp = "amp",
   AmpStory = "ampstory",
   Amp4Ads = "amp4ads",
   Amp4Email = "amp4email",
   PageExperience = "pageexperience",
-  Sxg = "sxg",
```

Comment:
should [`es5`](https://github.com/ampproject/amp-toolbox/blob/cff4fc0f1c5548e1b39d15d737ad530ee22459dd/prettier.config.js#L21) be changed to [`none`](https://prettier.io/docs/en/options.html#trailing-commas)?